language/stdlib/modules/libra_account.mvir

```diff
@@ -151,7 +151,7 @@ module LibraAccount {
     }
 
     // Withdraw `amount` LibraCoin.T from the transaction sender's account
-    public withdraw_from_sender(amount: u64): LibraCoin.T {
+    public withdraw_from_sender(amount: u64): LibraCoin.T acquires T {
```

Comment:
It seems a bit more readable to me if the `acquires` goes on the next line (particularly when there's a return value). But it's just syntax, so I don't feel strongly about this...

Reply:
I think if it didn't fit on one line, I'd agree. But no strong feelings either